data/json/items/gun/9mm.json

```diff
@@ -402,5 +402,32 @@
       [ "46", [ "hk46mag", "hk46bigmag" ] ],
       [ "57", [ "fn57mag" ] ]
     ]
+  },
+  {
+    "id": "glock_17",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Glock 17",
+    "description": "Designed for professionals, the Glock 17 is aimed towards law-enforcement and experienced shooters.",
+    "weight": 595,
+    "volume": 2,
+    "price": 69000,
+    "to_hit": -2,
+    "bashing": 8,
+    "material": [ "plastic", "steel" ],
+    "symbol": "(",
```

Comment:
This line is copied from gun_base/pistol_base already so it can be removed.

Reply:
material & symbol?

Reply:
"copy-from": "pistol_base", copies from the pistol_base item. Look in that abstract item. (Abstract means it isn't really real) and see which lines that you shouldn't copy here. Copy from copies them automatically. (Obviously).

Reply:
It's finished!